openapi/tests/openapi_integration/test_group.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from .helpers.helpers import request_with_validation
+from .helpers.collection_setup import basic_collection_setup, drop_collection
+
+collection_name = 'test_collection_reco'
+
+def upsert_chunked_docs(collection_name, docs=50, chunks=5):
+    points = []
+    for doc in range(docs):
+        for chunk in range(chunks):
+            doc_id = f"doc_{doc}"
+            i = doc * chunks + chunk
+            p = {"id": i, "vector": [0.5, 0.5, 0.5, 0.5], "payload": {"docId": doc_id}} 
```

Comment:
what do you think about introducing a few points where docID as different types:
- numbers
- string
- arrays

As a sanity check that the API is handling heterogeneous grouping key.

Reply:
Great idea, will do